src/applications/claims-status/components/StemAskVAQuestions.jsx

```diff
@@ -37,11 +36,11 @@ function StemAskVAQuestions() {
       <br />
       <p>Education Call Center:</p>
       <p>
-        <va-telephone contact={CONTACTS.GI_BILL} />
+        <va-telephone contact={CONTACTS.GI_BILL} uswds="false" />
         <span id="inside-US-tele">(inside the U.S.)</span>
       </p>
       <p>
-        <va-telephone contact="19187815678" international />
+        <va-telephone contact="9187815678" international uswds="false" />
```

Comment:
`va-telephone` doesn't have a `uswds` property

Reply:
yes so even though it doesnt have it yet, we still wanted to add it and set it to false since all components are supposed to be updated to V3 by feb 16.